apps/rush-lib/src/cli/actions/UpdateCloudCredentials.ts

```diff
@@ -78,8 +78,12 @@ export class UpdateCloudCredentials extends BaseRushAction {
           `If the ${this._deleteFlag.longName} is provided, no other parameters may be provided.`
         );
         throw new AlreadyReportedError();
+      } else if (buildCacheConfiguration.cloudCacheProvider) {
+        await buildCacheConfiguration.cloudCacheProvider.deleteCachedCredentialsAsync(terminal);
       } else {
-        await buildCacheConfiguration.cacheProvider.deleteCachedCredentialsAsync(terminal);
+        terminal.writeLine(
+          'No cloud build cache is configured. No credentials are stored and can be deleted.'
```

Comment:
Phrasing feels... odd?

Reply:
Agreed -- `No credentials are stored and can be deleted` grammatically sounds like `!(credentials are stored && credentials can be deleted)`, so it's logically possible that `credentials are stored but cannot be deleted`.

```suggestion
          'There is nothing to delete because no cloud build cache is configured..'
```